src/SamplesApp/SamplesApp.UITests/TestFramework/IsExternalInit.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Runtime.CompilerServices;
 
-#if !NET5_0 && !NET6_0
+#if !NET5_0 && !NET6_0_OR_GREATER
 #pragma warning disable CS0436
```

Comment:
```suggestion
```

Reply:
Indeed, I copy the file coming from uno.extensions, but this is actually not needed